### DISCUSSION TITLE
Welcome to ngods-stocks Discussions!

### DISCUSSION THREAD

**Discussion by zsvoboda** [zsvoboda/ngods-stocks#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BariSeba** [comment]
Hi, 
first of all: thank you very much for sharing your work. This is absolutely awesome and I'm excited to see where this will be going. I'm currently doing some research on a data platform architecture for the company I recently started at as a data engineer.
As I have to go the self hosted OSS route I came up with quite a similar stack. Definitely like the idea of combining dbt-trino-iceberg.
What are your thoughts on integrating Meltano/Airbyte to this stack?
I tried Airbyte as it is hyped at the moment but found it painfully slow. Meltano seems to be more promising from this perspective.
What are your thoughts on this?

**Reply by zsvoboda** [reply]
Hi,

It looks like we are doing quite similar work! I don’t have much experience with Meltano or Airbyte. I think that despite both companies are trying to reposition themselves as a broader stack (like AirFlow), their roots are in moving data (E and L part of the ELT/ETL process). Both vendors look almost same to me. Airbyte has perhaps more modern architecture while Meltano is based on a legacy Python framework (Singer) that has more connectors available.

I still didn’t need any connectors as I usually start with existing databases or structured files. When I need to extract data from some API or an exotic data source, I use a Dagster connector. Sometimes I need to write it myself.

I’d recommend you to take a look at an orchestration platforms that usually have many connectors included. AirFlow has the broadest support. You can take a look at Dagster or Prefect too.

Do you plan to use some data QA framework? I think that Airbyte and Meltano are adding some data testing capabilities. I don’t know any good open source framework for this. LMK if you know something.

Best,

ZD
